packer/install-dependencies.sh

```diff
@@ -126,6 +126,21 @@ for NVIDIA_DEVICE_PLUGIN_VERSION in ${NVIDIA_DEVICE_PLUGIN_VERSIONS}; do
     pullContainerImage "docker" "nvidia/k8s-device-plugin:${NVIDIA_DEVICE_PLUGIN_VERSION}"
 done
 
+TUNNELFRONT_VERSIONS="v1.9.2-v4.0.4"
```

Comment:
// TODO: cleanup images not needed at runtime in https://github.com/Azure/acs-engine/blob/cleanup-images/parts/k8s/kubernetesinstalls.sh#L220